### PR TITLE
[dom-gpu] magma 2.4

### PR DESCRIPTION
**Pull request by ajocksch**

### CODE REVIEW

**Comment by jgphpc** [comment]
add gnu version ?

File: easybuild/easyconfigs/m/magma/magma-2.4.0-CrayIntel-18.08-cuda-9.1.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+patches = [('make.inc-2.4.0', './')]
+dependencies = [ 
+  ('cudatoolkit/%s.85_3.18-6.0.7.0_5.1__g2eb7c52' % cudaversion, EXTERNAL_MODULE),
+]
+
+files_to_copy = ["lib", "include", "testing", "docs", "README", (['control/*.h'], 'control')]
+
+modextravars = { 'LD_LIBRARY_PATH':'$::env(CRAY_LD_LIBRARY_PATH):$::env(LD_LIBRARY_PATH)'}
+
+prebuildopts = 'mv make.inc-2.4.0 make.inc && CUDADIR=/opt/nvidia/cudatoolkit9.1/default '
```
Comment (gppezzi) [inline easybuild/easyconfigs/m/magma/magma-2.4.0-CrayIntel-18.08-cuda-9.1.eb]: You should use `$EBROOTCUDA` instead of hardcoding the path here

**Comment by ajocksch** [comment]
magma is only used by sirius at the moment (?) which is compiled with Intel

File: easybuild/easyconfigs/m/magma/magma-2.4.0-CrayIntel-18.08-cuda-9.1.eb (hunk 1 above)
Comment (ajocksch) [reply easybuild/easyconfigs/m/magma/magma-2.4.0-CrayIntel-18.08-cuda-9.1.eb]: done

File: easybuild/easyconfigs/m/magma/magma-2.4.0-CrayIntel-18.08-cuda-9.1.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+with current Multicore+GPU systems."""
+
+toolchain = {'name': 'CrayIntel', 'version': '18.08'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://icl.cs.utk.edu/projectsfiles/magma/downloads/']
+patches = [('make.inc-2.4.0', './')]
+dependencies = [ 
+  ('cudatoolkit/%s.85_3.18-6.0.7.0_5.1__g2eb7c52' % cudaversion, EXTERNAL_MODULE),
```
Comment (jgphpc) [inline easybuild/easyconfigs/m/magma/magma-2.4.0-CrayIntel-18.08-cuda-9.1.eb]: https://madra.cscs.ch/scs/doc/issues/355

Comment (ajocksch) [reply easybuild/easyconfigs/m/magma/magma-2.4.0-CrayIntel-18.08-cuda-9.1.eb]: please be more detailed; what is the problem here

Comment (jgphpc) [reply easybuild/easyconfigs/m/magma/magma-2.4.0-CrayIntel-18.08-cuda-9.1.eb]: https://github.com/eth-cscs/production/wiki/UES-standards-for-EB-recipes

Comment (jgphpc) [reply easybuild/easyconfigs/m/magma/magma-2.4.0-CrayIntel-18.08-cuda-9.1.eb]: Do you prefer not to use libsci_acc ?

Comment (jgphpc) [reply easybuild/easyconfigs/m/magma/magma-2.4.0-CrayIntel-18.08-cuda-9.1.eb]: https://github.com/eth-cscs/production/wiki/UES-standards-(CUDA,-Python,-...)-for-EB-recipes

**Comment by jgphpc** [comment]
exceeded the quota
retest this please

**Comment by vkarak** [comment]
retest this please

**Comment by jgphpc** [comment]
retest this please

**Comment by vkarak** [comment]
@jgphpc @ajocksch There seems to be a problem with the recipe. I've retested just before.

**Comment by teojgo** [comment]
It was failing due to the wrong [daint-gpu] title.

**Pull request merged**